ui/src/app/views/workspace-edit.tsx

```diff
@@ -130,34 +130,57 @@ export const researchPurposeQuestions = [
 export const specificPopulations = [
   {
     label: 'Race/Ethnicity',
-    object: SpecificPopulationEnum.RACEETHNICITY
+    object: SpecificPopulationEnum.RACEETHNICITY,
+    ubrLabel: 'Ancestry (Race/Ethnicity)',
+    ubrDescription: 'American Indian and Alaska Native (AIAN); Black, African American, or \
+       African; Middle Eastern or North African (MENA); Native Hawaiian or Other Pacific \
```

Comment:
nit: I would probably use + continuation, `\` should be injecting the indented spaces as part of the string. It probably renders normally as HTML, but feels a little wrong to depend on that.